scalafmt-tests/src/test/resources/test/Enum.stat

```diff
@@ -0,0 +1,51 @@
+runner.dialect = dotty
+<<< enum
+enum Color {
+case Red, Green, Blue
+}
+>>>
+enum Color {
+  case Red, Green, Blue
+}
+<<< parameterized enum
+enum Color (val rgb: Int) {
+case Red   extends Color(0xff0000)
+case Green extends Color(0x00ff00)
+case Blue  extends Color(0x0000ff)
+}
+>>>
+enum Color (val rgb: Int) {
```

Comment:
Shouldn't there be no space before opening paren?

Reply:
Good point, there is no space after the class identifier

Reply:
done